db/migrate/20210604120323_create_users.rb

```diff
@@ -0,0 +1,23 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :access_token
```

Comment:
@apeksha2697 please add unique indexing on git_id and login as you are using, validates_uniqueness_of :login, :git_id on user model.

Also, please think if email will be unique or not ?
->  if it'll also be unique then add unique indexing on email also
-> if not, then add normal indexing on email

Reply:
added unique indexing on email, login and git_id